test/unit/test_cli.py

```diff
@@ -72,12 +84,18 @@ def test_detector_and_image_queries():
     assert completed_process.returncode == 0
 
 
-def test_help():
+def test_help(no_api_token):
```

Comment:
Not a big deal, but a slightly simpler alternative to the fixture approach would be to use `mock.patch.dict()`

```python
import os
from unittest.mock import patch

@patch.dict(os.environ, {'GROUNDLIGHT_API_TOKEN': None})
def test_help():
```